<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="username">
     <property name="styleSheet">
      <string notr="true">color: blue; font: bold</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter text...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UserForm</sender>
   <signal>send(QColor,QString,QString)</signal>
   <receiver>UserForm</receiver>
   <slot>addText(QColor,QString,QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>send(QColor,QString,QString)</signal>
  <slot>addText(QColor,QString,QString)</slot>
 </slots>
</ui>
